amazon-kinesis-client/src/main/java/software/amazon/kinesis/coordinator/DefaultDiagnosticEventHandler.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 Amazon.com, Inc. or its affiliates.
+ * Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package software.amazon.kinesis.coordinator;
+
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import software.amazon.kinesis.annotations.KinesisClientInternalApi;
+
+@NoArgsConstructor
+@Slf4j
+@KinesisClientInternalApi
+public class DefaultDiagnosticEventHandler implements DiagnosticEventHandler {
```

Comment:
Avoid using the terms `Default`, `Basic`, etc. It is not clear on what the behavior of the class is. Also could you add javadocs.

Reply:
Changed name to DiagnosticEventLogger and added javadocs